packages/sdks/snippets/react/src/components/customTabsInfo.ts

```diff
@@ -0,0 +1,60 @@
+import CustomTabs from './CustomTabs';
+
+export const customTabsInfo = {
+  component: CustomTabs,
+  name: 'TabFields',
+  /** To accept children in your custom component and by default it is false */
+  canHaveChildren: true,
```

Comment:
`canHaveChildren` should not be set for components that use `<Blocks>`. It should only be set to `true` for components that use `props.children`

Reply:
Otherwise you can drag'n'drop components that end up nowhere. See https://www.loom.com/share/5951d5e1f03649139e06bb6043899446?from_recorder=1&focus_title=1